docs/_includes/calendar.html

```diff
@@ -0,0 +1,223 @@
+                </svg>
+            </button>
+        </div>
+    </div>
+    <div class="calendar__body">
+        <div class="calendar__month">
+            <!-- month table (static or interactive) -->
+        </div>
+        <div class="calendar__month">
+            <!-- month table (static or interactive) -->
+        </div>
+        <div class="calendar__month">
+            <!-- month table (static or interactive) -->
+        </div>
+    </div>
+</div>
+    {% endhighlight %}
+
+    <h3>Calendar Dropdown</h3>
+
+    <p>Calendars will stretch to the size of any container that they are placed inside, but the standard dropdown container should be used for most cases.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <button variant="icon" class="icon-btn icon-btn--small" aria-label="Open calendar" onclick="this.nextElementSibling.toggleAttribute('hidden')">
+                <svg aria-hidden="true" class="icon icon--calendar" focusable="false">
+                    {% include symbol.html name="calendar" %}
+                </svg>
+            </button>
+            <div class="calendar__dropdown" hidden>
+                <div class="calendar">
+                    <div class="calendar__header">
+                        <div class="calendar__header--inner">
+                            <button variant="icon" class="icon-btn icon-btn--small icon-btn--transparent" aria-label="Previous Month - July 2019">
+                                <svg aria-hidden="true" class="icon icon--chevron-left" focusable="false">
+                                    {% include symbol.html name="chevron-left" %}
+                                </svg>
+                            </button>
+                            <h3>August 2019</h3>
+                            <h3>September 2019</h3>
+                            <button variant="icon" class="icon-btn icon-btn--small icon-btn--transparent" aria-label="Next Month - October 2019">
+                                <svg aria-hidden="true" class="icon icon--chevron-right" focusable="false">
+                                    {% include symbol.html name="chevron-right" %}
+                                </svg>
+                            </button>
+                        </div>
+                    </div>
+                    <div class="calendar__body">
+                        <div class="calendar__month">
+                            {% include calendar-month.html index="1" interactive="true" %}
+                        </div>
+                        <div class="calendar__month">
+                            {% include calendar-month.html index="2" interactive="true" %}
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    {% highlight html %}
+<button variant="icon" class="icon-btn icon-btn--small" aria-label="Open calendar" onclick="this.nextElementSibling.toggleAttribute('hidden')">
```

Comment:
This example feels odd because we shouldn't have an element class like `calendar__dropdown` exist outside of a block class. We'd discussed having this all wrapped inside of a `calendar-button` block.

Reply:
Another option might be to rename `calendar__dropdown` to something like `calendar-popover`...

Reply:
My issue with this is that I don't think `calendar-button` should be a direct option on eBayUI, since the calendar should always be associated with a date input. From my perspective, we _shouldn't_ have full control over the button because we can use other components that we've built in (in this example I used `icon-button`). I do like the idea of adding `calendar-popover`, though it would be a _very_ barebones component.

Reply:
> since the calendar should always be associated with a date input

Not necessarily always. Don't we have calendar placed directly in a modal? There's no date input there.

> we shouldn't have full control over the button

We do this with listbox-button and menu-button, so there's something to be said for consistency too. The idea would be that the calendar-button is placed inside of the textbox as the postfix button. We could even demonstrate this in the textbox section.

Reply:
>> since the calendar should always be associated with a date input

> Not necessarily always. Don't we have calendar placed directly in a modal? There's no date input there.

Yea, I agree with Ian here. We talked before about the calendar being a completely independent thing from any sort of input. There are various implementation possibilities, even one that is a completely static variation that simply has the calendar directly on a page.

Reply:
We've sort of discussed this via Slack, but not completely. I think the current implementation where `date-input` is a separate component from `calendar` works pretty well in this case, because it sort of sidesteps the issue by saying that _calendars_ can be used in any circumstance, but _date inputs_ must be associated with a textbox.